app/containers/Home/Home.jsx

```diff
@@ -1,50 +1,143 @@
+        <div className={styles.loginText}>Login</div>
+
+        <div className={styles.inputContainer}>
+          <SelectInput
+            className={styles.input}
+            onChange={value => this.handleSelect(value)}
+            value={this.state.option}
+            items={['Private Key']}
+            getItemValue={() => this.state.option}
+          />
+
+          <TextInput
+            className={styles.input}
+            onChange={(e: Object) =>
+              this.setState({ password: e.target.value })
+            }
+            placeholder="Password"
+            type="password"
+          />
+
+          <Button
+            renderIcon={() => <Login />}
+            icon="login"
+            style={{ marginTop: 20 }}
+            onClick={() => this.handleSubmit()}
+          >
+            Login
+          </Button>
+
+          <div className={styles.buttonRow}>
+            <div style={{ flex: 0.45 }}>
+              <Button primary renderIcon={() => <Plus />}>
+                New Wallet
+              </Button>
+            </div>
+            <div style={{ flex: 0.45 }}>
+              <Button primary renderIcon={() => <Wallet />}>
+                Wallet Manager
+              </Button>
+            </div>
+          </div>
+        </div>
       </div>
-    </Link>
-  </div>
-)
+    </div>
+  )
+}
 
 export default Home
+
+// // @flow
+// import React from 'react'
+// import classNames from 'classnames'
+// import { Link } from 'react-router-dom'
+//
+// import { ROUTES } from '../../core/constants'
+//
+// import styles from './Home.scss'
+//
+// const Home = () => (
+//   <div id='home'>
+//     <Link to={ROUTES.LOGIN_LOCAL_STORAGE}>
+//       <div className={classNames('linkBox', styles.linkBox)}>
+//         Login using a saved wallet
+//       </div>
+//     </Link>
+//     <Link to={ROUTES.LOGIN_NEP2}>
+//       <div className={classNames('linkBox', styles.linkBox)}>
+//         Login using an encrypted key
+//       </div>
+//     </Link>
+//     <Link to={ROUTES.LOGIN_PRIVATE_KEY}>
+//       <div className={classNames('linkBox', styles.linkBox)}>
+//         Login using a private key
+//       </div>
+//     </Link>
+//     <Link to={ROUTES.LOGIN_LEDGER_NANO_S}>
+//       <div className={classNames('linkBox', styles.linkBox)}>
+//         Login using a Ledger
+//       </div>
+//     </Link>
+//     <Link to={ROUTES.CREATE_WALLET}>
+//       <div className={classNames('linkBox', styles.linkBox, styles.linkBoxAlt)}>
+//         Create a new wallet
+//       </div>
+//     </Link>
+//     <Link to={ROUTES.ENCRYPT_KEY}>
+//       <div className={classNames('linkBox', styles.linkBox, styles.linkBoxAlt)}>
+//         Encrypt an existing key
+//       </div>
+//     </Link>
+//     <Link to={ROUTES.SETTINGS}>
+//       <div className={classNames('linkBox', styles.linkBox, styles.linkBoxAlt)}>
+//         Manage Neon settings
+//       </div>
+//     </Link>
+//   </div>
```

Comment:
It looks like these routes need to go somewhere still.  Will merging this break logging in and seeing the rest of the app?  I think we should at least ensure routing works before merging.  (I can help if needed.)